### ISSUE TITLE
Not working with jQuery 3.0.0 and higher

### ISSUE THREAD

**Issue by xahon** [malihu/malihu-custom-scrollbar-plugin#489]
http://jsfiddle.net/oy5uwk8y/18/

If you change jQuery ver to lower it does work

**Comment by malihu** [comment]
This is because you're using `$(window).load( function() {` in your script. Change it to: 
`$(window).on("load", function() {`

See similar issue: https://github.com/malihu/malihu-custom-scrollbar-plugin/issues/460

**Comment by Mmatiasn** [comment]
I'm having issues getting it to work. I'm not even getting errors.

```
/* addons.js */
(function ($) {
    "use strict";
    
    $(window).on("load", function () {

        $("body").mCustomScrollbar({
            theme: "inset-3-dark"
        });

    });

}(jQuery));
```

I've also tried putting it on a small div and it's not working at all.

`#smallDiv`

This is the versions I'm using

- jQuery JavaScript Library v3.1.1
- malihu jquery custom scrollbar plugin == Version: 3.1.5

**Comment by malihu** [comment]
@Mmatiasn The plugin should work with jQuery 3.x. Try creating a jsfiddle or a test page so I can see what happens and help.

**Comment by Praveen-raj27** [comment]
I have updated the JQuery to 3.6.0 and this mCustomScrollbar is not working in my project. I have also checked by using JS Fiddle. mCustomscrollbar is working till 3.3.1 
http://jsfiddle.net/lustre/2ozyvmbw/
Can you help me to fix it in JQuery 3.6.0?

**Comment by gowtham758550** [comment]
@Praveen-raj27 do you find any other way to use mCustomscrollbar with jquery3.6
